cmd/build/helmify/replacements.go

```diff
@@ -114,6 +114,51 @@ var replacements = map[string]string{
 
 	"- HELMSUBST_MUTATION_STATUS_ENABLED_ARG": `{{ if not .Values.disableMutation}}- --operation=mutation-status{{- end }}`,
 
+	"- HELMSUBST_CONTROLLER_MANAGER_OPERATIONS": `
+        {{- if not .Values.controllerManager.disableWebhookOperation }}
+        - --operation=webhook
+        {{- end }}
+        {{- if not .Values.controllerManager.disableGenerateOperation }}
+        - --operation=generate
+        {{- end }}
+        {{- if not .Values.controllerManager.disableAuditOperation }}
+        - --operation=audit
+        {{- end }}
+        {{- if not .Values.controllerManager.disableStatusOperation }}
+        - --operation=status
+        {{- end }}
+        {{- if not .Values.controllerManager.disableMutationStatusOperation }}
+        - --operation=mutation-status
+        {{- end }}
+        {{- if not .Values.controllerManager.disableMutationControllerOperation }}
+        - --operation=mutation-controller
+        {{- end }}`,
+
+	"- HELMSUBST_AUDIT_OPERATIONS": `
+        {{- if not .Values.audit.disableWebhookOperation }}
+        - --operation=webhook
```

Comment:
lets remove `webhook` operation from audit pod as just enabling `operation=webhook` on audit pod is not enough for validating admission requests in audit pod. This may lead to confusion in users as well as could lead to un intended behaviors.